src/SFA.DAS.FAT.Application/Locations/Services/LocationService.cs

```diff
@@ -11,18 +12,42 @@ public class LocationService : ILocationService
         private readonly IApiClient _client;
         private readonly FindApprenticeshipTrainingApi _config;
 
-        public LocationService (IApiClient client, IOptions<FindApprenticeshipTrainingApi> config)
+        public LocationService(IApiClient client, IOptions<FindApprenticeshipTrainingApi> config)
         {
             _client = client;
             _config = config.Value;
         }
-        public async Task<Domain.Locations.Locations> GetLocations(string searchTerm)
+        public async Task<Domain.Locations> GetLocations(string searchTerm)
         {
+            if (searchTerm.Trim().Length < 3) return new Domain.Locations();
             var request = new GetLocationsApiRequest(_config.BaseUrl, searchTerm);
+            return await _client.Get<Domain.Locations>(request);
+        }
+
+        public async Task<bool> IsLocationValid(string locationName)
+        {
+            locationName = locationName.Trim();
+
+            if (locationName.Contains(','))
+            {
+                var firstItem = locationName.Split(',').First().Trim();
+                if (firstItem.Length < 3) return false;
+                var locations = await GetLocations(firstItem);
+                return locations != null && locations.LocationItems.Any(x => x.Name == locationName);
```

Comment:
the null check is unnecessary as the method has no path where it returns null

Reply:
removed